### ISSUE TITLE
Middleman Command cannot be found, dependencies do not install.

### ISSUE THREAD

**Issue by geoffdavis92** [middleman/middleman#1941]
I have been having some issues with MM on my machine (Mac OS X Yosemite) and now trying to reinstall Middleman.

However, after installing my local `bundle` and manually installing the MM CLI, the `middleman` command still does not work.

I removed all related MM files/directories, and went into my team's MM project and ran `$ bundle install`. Here's the output:

<pre>
Using i18n 0.7.0  
Using json 1.8.3  
Using minitest 5.7.0  
Using thread_safe 0.3.5  
Using hitimes 1.2.2  
Using chunky_png 1.3.4   
Using coffee-script-source 1.9.1.1  
Using execjs 2.5.2  
Using multi_json 1.11.1  
Using sass 3.4.15  
Using rb-fsevent 0.9.5  
Using ffi 1.9.8
Using eventmachine 1.0.7
Using http_parser.rb 0.6.0
Using erubis 2.7.0
Using tilt 1.4.1
Using hike 1.2.3
Using uber 0.0.13
Using kramdown 1.7.0
Using bundler 1.12.5
Using rack 1.6.4
Using thor 0.19.1
Using tzinfo 1.2.2
Using timers 4.0.1
Using coffee-script 2.4.1
Using uglifier 2.7.1
Using compass-core 1.0.3
Using compass-import-once 1.0.5
Using rb-inotify 0.9.5
Using em-websocket 0.5.1
Using haml 4.0.6
Using hooks 0.4.0
Using rack-test 0.6.3
Using sprockets 2.12.3
Using rack-livereload 0.3.15
Using activesupport 4.1.11
Using celluloid 0.16.0
Using compass 1.0.3
Using sprockets-helpers 1.1.0
Using sprockets-sass 1.3.1
Using padrino-support 0.12.5
Using listen 2.10.1
Using padrino-helpers 0.12.5
Using middleman-core 3.3.12
Using middleman-sprockets 3.4.2
Using middleman-livereload 3.1.1
Using middleman 3.3.12
Bundle complete! 4 Gemfile dependencies, 47 gems now installed.
Use `bundle show [gemname]` to see where a bundled gem is installed.</pre>

Looks like it works? I tried running `$ bundle exec middleman` because in the past simply running `$ middleman` gave me an error, but the bundle method worked. Here's what I got:
<pre>
bundler: command not found: middleman
Install missing gem executables with `bundle install`
</pre>

Weird, it looked like Middleman was installed, so I checked using `$ bundle show` and it showed me where it was: `/Library/Ruby/Gems/2.0.0/gems/middleman-3.3.12`.

I noticed that the bundle did not install the middleman CLI, so I installed that manually using `$ gem install middleman-cli`, which then successfully installed.

Again, I tried to run the MM executable, but got returned this error:
<pre>
bundler: failed to load command: middleman (/usr/bin/middleman)
LoadError: cannot load such file -- middleman-core/profiling
  /Library/Ruby/Gems/2.0.0/gems/middleman-cli-4.1.9/bin/middleman:3:in `require'
  /Library/Ruby/Gems/2.0.0/gems/middleman-cli-4.1.9/bin/middleman:3:in `<top (required)>'
  /usr/bin/middleman:22:in 'load'
  /usr/bin/middleman:22:in '&lt;top (required)>'
</pre>

Okay so I actually got somewhere that time. Before I cleared all my MM gems/directories out, I tried following this error path, manually `$ gem install`-ing each missing package from the "cannot load file" line, but after a few installs in, I noticed this was going to be burdensome, and seemed like a failure with MM.

So to recap, here are my issues:

* Upon installing my project's bundle, `middleman`/the CLI does not install or is not linking correctly
* After manually installing the middleman-cli gem, it appears that the MM dependencies are not also installing.
* Before tracing the error stack and reinstalling, I kept receiving errors notifying me that the `middleman` command cannot be found

**Comment by tdreyno** [comment]
Hope you were able to find a solution. Closing this, but maybe the forum will be able to help troubleshoot.

**Comment by geoffdavis92** [comment]
@tdreyno Yes, it must have been something with Ruby; I reinstalled the language and got it working again.
